src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_client_factory.py

```diff
@@ -21,7 +21,7 @@ def get_token(server, resource, scope):  # pylint: disable=unused-argument
         import adal
         from azure.cli.core._profile import Profile
         try:
-            return Profile(cli_ctx=cli_ctx).get_login_credentials(resource)[0]._token_retriever()  # pylint: disable=protected-access
+            return Profile(cli_ctx=cli_ctx).get_raw_token(resource)[0]
```

Comment:
I didn't write tests in this PR as i feel the existing testbed should be sufficient to capture potential regressions